### DISCUSSION TITLE
is possible atomWithRefresh can refresh another atom?

### DISCUSSION THREAD

**Discussion by ramunecroft** [pmndrs/jotai#2390] [Q&A]
```typescript
export const loadTasksAtom = atom(null, async (get, set) => {
  try {
    const tasksResponse = await getTasks();
    console.log("tasksResponse: ", tasksResponse);
    if (tasksResponse?.status === 200) {
      console.log("is Success");
      const json = (await tasksResponse.json()) as TaskType[];
      set(messageAtom, "Tasks loaded successfully.");
      set(taskListAtom, json);
    } else {
      set(messageAtom, "Failed to load tasks.");
    }
  } catch (error) {
    console.log(error);
  }
});

export const refresher = atomWithRefresh(get => {
  return get(loadTasksAtom);
});

```
I have some code that sets messages based on the success of an API call within Atom. 

However, it seems that atomWithRefresh only supports get operations and doesn't allow for internal set actions.

 Is it possible to combine these two functionalities?

**Comment by dai-shi** [comment]
I'm not sure if I understand the goal, but **if** it's about write-only atoms and enabling suspense, here's my blog post: https://blog.axlight.com/posts/how-to-use-jotai-and-use-transition-for-mutation/

**Reply by ramunecroft** [reply]
Cool, 
I could solve my issue, the post really helped me 
Thanks.
